demo/psa/prov-verif-e2e/README.md

```diff
@@ -0,0 +1,156 @@
+# PSA Demonstration Steps
+
+This document explains all the steps one need to follow to successfully run
+end to end PSA Demonstration
+
+## Preconditions
+
```

Comment:
Would be great to have here all the prerequisites install instructions (mockgen, protoc) like in this demo branch:
https://github.com/veraison/enact-demo/tree/main/docs/veraison-build

Reply:
Thank you and I agree, we need to add this!